charts/newrelic-k8s-metrics-adapter/tests/job_patch_clusterrole_test.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.rbac.pspEnabled }}
```

Comment:
You cannot use templating in test files, it's not going to work. The only test that fails is `common_extra_naming_test.yaml` as it's checking the name is properly defined. We can extend it to set the `pspEnabled` default value like:
```
  - it: default values has its name correctly defined
    set:
      personalAPIKey: 21321
      config:
        accountID: 11111111
      rbac:
        pspEnabled: true
```